first complete draft of `Jupyter book` (#74)
neuroscout/neuroscout-paper: paper/intro.md

[![GitHub issues](https://img.shields.io/github/issues/neuroscout/neuroscout-paper?style=plastic)](https://github.com/neuroscout/neuroscout-paper/issues)
[![GitHub PR](https://img.shields.io/github/issues-pr/neuroscout/neuroscout-paper)](https://github.com/neuroscout/neuroscout-paper/pulls)
[![License](https://img.shields.io/github/license/neuroscout/neuroscout-paper)](https://github.com/neuroscout/neuroscout-paper)

## Welcome
Welcome! This is an interactive walkthrough of our publication “Neuroscout: a unified platform for generalizable and reproducible fMRI research”. Here you can visualize and re-run the code we used to create analyses and figures. 

In the paper, we validate the platform through some replication meta-analyses, and we showcase some more exploratory  applications that demonstrate how Neuroscout can be used to run larger-scale, more generalizable naturalistic fMRI research.

These analyses require specifying and estimating models at the level of individual datasets/tasks, and the outputs of these analyses are used as inputs to meta-analyses. This is reflected by the structure of the GitHub repository and of this book.

## Rerunning the analyses

> The `analyses` follow the structure of figures in the manuscript. Most `analyses` **require** first running `single dataset results` and then performing a `meta-analysis`.

Reviewer: ```suggestion
The `analyses` follow the structure of figures in the manuscript. Most `analyses` **require** first running `single dataset results` (using `Neuroscout`) and then performing a `meta-analysis` (using `NiMARE`)
```

You can use this resource to simply visualize the analyses, or to rerun them and recreate the figure.

> Note that, if you want to rerun meta-analyses, you do not need to rerun the dataset-level models. All statistical maps are uploaded to `NeuroVault` and can be downloaded using our meta-analysis code. If you wish to recreate and re-estimate dataset-level models, you will have to do so locally.

Reviewer: ```suggestion
Note that, if you want to re-run meta-analyses, you do not need to rerun the dataset-level models. All statistical maps are uploaded to `NeuroVault` and can be downloaded using our meta-analysis code. If you wish to recreate and re-estimate dataset-level models, you will have to do so locally.
```

- `Cloud computing`

>    Notebooks can be rerun on the cloud using [mybinder](https://mybinder.org/) by clicking on the `rocket` icon at the top of the notebook page. This is potentially the easiest option as you don't have to install/download anything. You can also easily access all analyses at:

Reviewer: ```suggestion
   Notebooks can be re-run on the cloud using [mybinder](https://mybinder.org/) by clicking on the `rocket` icon at the top of the notebook page. This is potentially the easiest option as you don't have to install/download anything. You can also easily access all analyses at:
```
